src/Components/Components/src/Layouts/LayoutAttribute.cs

```diff
@@ -10,14 +9,8 @@ namespace Microsoft.AspNetCore.Components.Layouts
     /// Indicates that the associated component type uses a specified layout.
     /// </summary>
     [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
-    public class LayoutAttribute : Attribute
+    public sealed class LayoutAttribute : Attribute
```

Comment:
I didn't move the namespace of this type in THIS pr because it will require a compiler update.